WordPress-Theme/ruleset.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- For more information: https://make.wordpress.org/themes/handbook/review/ -->
 	<description>Standards any Theme to be published on wordpress.org should comply with.</description>
 
-
+	<rule ref="WordPress.Theme.DeregisterScript"/>
```

Comment:
No need to specify the specific rule. All sniffs in the `Theme` folder will be automatically added once the below has gone into the ruleset (and it will as soon as we merge the first custom sniff).
```xml
<rule ref="WordPress.Theme/>
```